core/trino-main/src/main/java/io/trino/operator/partition/PartitionedOutputOperator.java

```diff
@@ -11,7 +11,7 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-package io.trino.operator;
+package io.trino.operator.partition;
```

Comment:
I would call the package `output` and move `TaskOutputOperator` (+ tests)` there too